h2o-genmodel/src/main/java/hex/genmodel/GenModel.java

```diff
@@ -190,6 +190,10 @@ public int getOrigNumCols() {
     return null;
   }
 
+  /** Returns original Eigen encoder projections array for all columns. */
+  @Override
+  public double[] getOrigProjectionArray() {return new double[]{0};}
```

Comment:
why is this here? I do not see it implemented/overriden anywhere

Reply:
it is used in SharedTreeModel.java#toJavaInit() line 666, to pass array which defines the eigen encoder to GenModel

Reply:
👌 changed to return null